environments/demo/backend_lb_config.yaml

```diff
@@ -321,4 +321,12 @@ gateways:
       - product: rd
         component: location-ref-api-int
       - product: rd
-        component: commondata-api
\ No newline at end of file
+        component: commondata-api
+        
+    # HMC
+      - product: hmc
+        component: hmc-cft-hearing-service
```

Comment:
you prob don't want this doubled up, the url would be `hmc-hmc-cft-hearing-service`

Reply:
Ah of course, thanks @timja - adjusted!